ci/prow/make_config.go

```diff
@@ -299,6 +301,51 @@ func getMapSlice(m interface{}) yaml.MapSlice {
 	return nil
 }
 
+func combineSlices(a1 []string, a2 []string) []string {
+	var res []string
+	res = append(res, a1...)
+	for _, e2 := range a2 {
+		add := true
+		for _, e1 := range a1 {
+			if e1 == e2 {
+				add = false
+			}
+		}
+		if add {
+			res = append(res, e2)
+		}
+	}
+	return res
+}
+
+// Consolidate approved and skipped branches with newly added approved/skipped
```

Comment:
"approved" branches is a concept hard to understand for someone not authoring this PR...